doc/contributing/collaborator-guide.md

```diff
@@ -540,6 +540,12 @@ For pull requests from first-time contributors, be
 [welcoming](#welcoming-first-time-contributors). Also, verify that their git
 settings are to their liking.
 
+If the pull request contains more than one commit, it can be landed either by
+squashing into one commit or by rebasing all the commits. Generally, a
+collaborator should land the pull request with squashing. If the pull request
+has more than one self-contained subsystem commits, a collaborator may land it
+with rebasing.
```

Comment:
Maybe we should clarify what "rebasing all the commits" means, as we usually rebase with `--autosquash` – at least the CI does, folks manually landing PRs have free reign and they would typically squash some related commits together. Also, if the PR contains special commits (see https://github.com/nodejs/node-core-utils/pull/710), you'd still want to rebase despite wanting the PR to land as one commit.

```suggestion
If the pull request contains more than one commit, it can be landed either by
squashing into one commit, by rebasing all the commits, or a mix of the two.
Most pull requests should land as one commit. If the pull request
has more than one self-contained subsystem commit, it can land as several commits.
```

Reply:
Thanks, this sounds good! Updated.